docs/_interface/sync-modes.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Sync-modes
+sort-key: L
+---
+
+Syncing is the process by which Geth catches up to the latest Ethereum block and current global state. 
+There are several ways to sync a Geth node that differ in their speed, storage requirements and trust 
+assumptions. This page outlines three sync configurations for full nodes and one for light nodes.
+
+## Full nodes
+
+There are two types of full node that use different mechanisms to sync up to the head of the chain:
+
+### Snap (default)
+
+A snap sync'd node holds the most recent 128 block states in memory, so transactions in that range are always quickly 
+accessible. However, snap-sync only starts processing from a relatively recent block (as opposed to genesis 
+for a full node). Between the initial sync block and the 128 most recent blocks, the node stores occasional 
+checkpoints that can be used to rebuild the state on-the-fly. This means transactions can be traced back as 
+far as the block that was used for the initial sync. Tracing a single transaction requires reexecuting all 
+preceding transactions in the same block **and** all preceding blocks until the previous stored snapshot. 
+Snap-sync'd nodes are therefore functionally equal to full nodes, but the initial synchronization required 
```

Comment:
A note about terminology.. A node is either a full node or a light node. 
Therefore, a snap-synced node is a full node. 
An `syncmode=full` is also a fulll node. 
So it' snot "functionally equal to full nodes". They are full nodes.

Reply:
```suggestion
Snap-sync'd nodes are therefore full nodes, with the only difference being the initial synchronization required 
```